baseframe/forms/validators.py

```diff
@@ -9,6 +9,7 @@
 from fractions import Fraction
 import datetime
 import re
+import emoji
```

Comment:
install `isort` and sort these imports. Or install `pre-commit` and let it do that.